### PR TITLE
CRD migration

### PR DESCRIPTION
**Pull request by yifan-gu**
- Bump tco, kvo, tpo to support CRD.
- Add tectonic-cluo.
- Change all TPR manifests to CRD manifests.

### CODE REVIEW

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by Quentin-M** [comment]
config.tf looks the same as my own rebase.
did you verify payload.json?

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by Quentin-M** [comment]
@yifan-gu does 3d18f45 contains the op-client fix for CRD overwrite?

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
@Quentin-M Not yet, waiting for tco e2e to pass, then I will cut another release.

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

File: config.tf [hunk 1]
```diff
@@ -25,43 +25,43 @@ variable "tectonic_container_images" {
...
+    pod_checkpointer             = "quay.io/coreos/pod-checkpointer:3517908b1a1837e78cfd041a0e51e61c7835d85f"
+    prometheus                   = "quay.io/prometheus/prometheus:v1.7.1"
+    prometheus_config_reload     = "quay.io/coreos/prometheus-config-reloader:v0.0.2"
+    prometheus_operator          = "quay.io/coreos/prometheus-operator:v0.11.1"
+    stats_emitter                = "quay.io/coreos/tectonic-stats:6e882361357fe4b773adbf279cddf48cb50164c1"
+    stats_extender               = "quay.io/coreos/tectonic-stats-extender:487b3da4e175da96dabfb44fba65cdb8b823db2e"
+    tectonic_channel_operator    = "quay.io/coreos/tectonic-channel-operator:0.5.2"
+    tectonic_etcd_operator       = "quay.io/coreos/tectonic-etcd-operator:v0.0.1"
+    tectonic_monitoring_auth     = "quay.io/coreos/tectonic-monitoring-auth:v0.0.1"
+    tectonic_prometheus_operator = "quay.io/coreos/tectonic-prometheus-operator-dev:477fa943"
```
Comment (Quentin-M) [inline config.tf]: @brancz just cut the actual 1.5.1 version (https://github.com/coreos/tectonic-installer/pull/1740#issuecomment-324256049)

```
quay.io/coreos/tectonic-prometheus-operator:v1.5.1
```

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by yifan-gu** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by Quentin-M** [comment]
test upgrade from 1.7.1-tectonic.2 1.7.1-tectonic.2

**Comment by Quentin-M** [comment]
- Smoke tests on some of the Azure matrix items failed due to quota. Bare-metal smoke test failed due to IP conflict. All other items passed.
- The last upgrade test passed successfully, waiting for another one before merging.

**Pull request merged**